spec/Dockerfile.omiserver

```diff
@@ -3,12 +3,22 @@
 FROM ubuntu:18.04
 
 ARG BOLT_PASSWORD=bolt
+# NOTE: Only designed to be set at build time, not at run time!
+ENV BOLT_PASSWORD=${BOLT_PASSWORD}
+ENV KRB5_REALM=
```

Comment:
Should these be getting set? Do they need to be here if not?

Reply:
They are passed in through the `docker-compose.yml` declaration of the `omiserver` service.